cypress/e2e/shared/about.test.ts

```diff
@@ -1,28 +1,39 @@
 import {Organization} from '../../../src/types'
 
 describe('About Page', () => {
+  const apiOrgPath =
```

Comment:
Can we also have IS_OSS const so that you don't have to do url check again in a couple of other places?

Reply:
Converted to a constant